packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -30,6 +30,9 @@ export interface SentryWrappedXMLHttpRequest extends XMLHttpRequest {
   };
 }
 
+// Dynamic keys like the one below doesn't work with interfaces :(
+// [key in InstrumentHandlerType]?: boolean;
+
 /** JSDoc */
 interface BreadcrumbIntegrations {
```

Comment:
Also, add depreate here, the name should become something like, InstrumentOptions